innopoints/views/account.py

```diff
@@ -65,7 +71,8 @@ def get_info(email):
         user = Account.query.get_or_404(email)
 
     out_schema = AccountSchema(exclude=('moderated_projects', 'created_projects', 'stock_changes',
-                                        'transactions', 'applications', 'reports'))
+                                        'transactions', 'applications', 'reports',
+                                        'notification_settings'))
```

Comment:
but why? I say they should be part of the returned account object, and the dedicated endpoint for `GET /account/notification_settings` is useless. You wouldn't ever request it except in the profile, which is where you request `/account` anyways.

Reply:
You request `/account` on many other occasions, to get balance, for instance. Figured it would be redundant information there.

Reply:
I just don't like that we have so many dedicated endpoints for properties inside an object, either for `GET` or `PATCH`. The notification settings are part of the account. If you request the account, you get everything in it. It is up to the front end to store in the local "store" and not re-request them.

Reply:
You cannot not rerequest them. If you want to keep the account data up to date, you have to request everything. And what's bad about having several endpoints? They all send serve their own role

Reply:
Debatable design decisions, but alright